### PR TITLE
Frontend support for SeedExplorer

### PR DESCRIPTION
**Pull request by andrewcode-create**
Includes a frontend page for the seed explorer, issue #44 . 
Pending #45 for an api to get a specific seed from the server.

Currently doesn't change the website, as the links are commented out.

### CODE REVIEW

**Comment by StefanOltmann** [comment]
"Seed explorer" could confuse people, because the seed browser is called "map explorer" and "map" & "seed" are basically synonyms. I believe we need a better name here.

@Sgt-Imalas I think you suggested "seed viewer" on Discord

**Comment by andrewcode-create** [comment]
Got it, will change it now

**Comment by andrewcode-create** [comment]
All done!

File: frontend/src/components/Navbar.vue [hunk 1]
```diff
@@ -12,6 +12,11 @@
                     <li class="nav-item">
                         <router-link to="/map-explorer" class="nav-link">Map Explorer</router-link>
                     </li>
+                    <!--
```
Comment (andrewcode-create) [inline frontend/src/components/Navbar.vue:15]: Un-comment this when the api is done

File: frontend/src/views/HomeView.vue [hunk 2]
```diff
@@ -13,7 +13,8 @@
     </p>
 
     <div class="d-flex gap-3 mt-5 flex-column flex-md-row">
-      <router-link to="/map-explorer" class="btn btn-lg btn-outline-primary flex-grow-1">Map Explorer</router-link>
+      <router-link to="/map-explorer" class="btn btn-lg btn-outline-warning flex-grow-1">Map Explorer</router-link>
+      <!--<router-link to="/seed-explorer" class="btn btn-lg btn-outline-primary flex-grow-1">Seed Explorer</router-link>-->
```
Comment (andrewcode-create) [inline frontend/src/views/HomeView.vue:17]: Uncomment this when the api is done

File: frontend/src/views/SeedViewerView.vue [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<script setup>
+import { ref } from 'vue';
+
+const seedInput = ref("0000-0-0-0-0");
+
+const submit = () => {
+    // TODO add api code here
```
Comment (andrewcode-create) [inline frontend/src/views/SeedViewerView.vue:7]: implement api and display the map here

**Pull request merged**